VimeoNetworking/VimeoNetworking/Request+Comment.swift

```diff
@@ -0,0 +1,26 @@
+//
+//  Request+Comment.swift
+//  Pods
+//
+//  Created by King, Gavin on 10/20/16.
+//
```

Comment:
Missing license header (unless I added it in my PR, but this looks like a new file)